services/ui-src/src/measures/2022/globalValidations/validateNoNonZeroNumOrDenom/index.ts

```diff
@@ -8,13 +8,14 @@ import {
 import {
   convertOmsDataToRateArray,
   getOtherPerformanceMeasureRateArray,
+  // getPerfMeasureRateArray,
 } from "../dataDrivenTools";
 
 interface ValProps extends UVFP {
   hybridData?: boolean;
 }
 
-const _validationRateNotZero = ({ location, rateData }: UVFP) => {
+export const _validationRateNotZero = ({ location, rateData }: UVFP) => {
```

Comment:
Is there a reason we're exporting this? The design pattern in our globalValidation functions is to have private `_validation` functions that do the actual validation check. They should be invoked by a specific PM or OMS function. This function `_validationRateNotZero` should never be called outside this file.
```suggestion
const _validationRateNotZero = ({ location, rateData }: UVFP) => {
```